lib/OpenQA/CacheService/Model/Downloads.pm

```diff
@@ -42,7 +42,8 @@ sub add {
 sub find {
     my ($self, $lock) = @_;
     return undef
-      unless my $hash = $self->sqlite->db->select('downloads', ['job_id'], {lock => $lock}, {-desc => 'id'})->hash;
+      unless my $hash
+      = $self->cache->sqlite->db->select('downloads', ['job_id'], {lock => $lock}, {-desc => 'id'})->hash;
```

Comment:
As this already extends over multiple lines I would do something like:

```perl
my @args = ('downloads', ['job_id'], {lock => $lock}, {-desc => 'id'});
return undef unless my $hash = $self->cache->sqlite->db->select(\@args)->hash;
```

and of course I have no clue if any of the "@" or "()" or "\@" is the right syntax here :D

Reply:
The syntax is wrong, the function doesn't expect an arrayref :-)

Reply:
To quote coolo: "I wrote pseudo-code" :)